src/Symfony/Component/RateLimiter/Rate.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\RateLimiter;
+
+/**
+ * Data object representing the fill rate of a token bucket.
+ *
+ * @author Wouter de Jong <wouter@wouterj.nl>
+ *
+ * @final
```

Comment:
I'd prefer to use `final class Rate` instead. Any reasons not to do so?

Reply:
I thought Symfony never used PHP's final. Fixed